Game.java

```diff
@@ -2,56 +2,73 @@
 
 public class Game{
 
-	List<bigPlayer> lobby = new ArrayList<bigPlayer>();
-	List<bigPlayer> team1 = new ArrayList<bigPlayer>();
-	List<bigPlayer> team2 = new ArrayList<bigPlayer>();
+	List<Player> lobby = new ArrayList<Player>();
+	List<Player> team1 = new ArrayList<Player>();
+	List<Player> team2 = new ArrayList<Player>();
+	List<Map> mapList = new ArrayList<Map>();
+	List<Player> team1MapPickers = new ArrayList<Player>();
+	List<Player> team2MapPickers = new ArrayList<Player>();
+	static Scanner in = new Scanner(System.in);
 
 	public static void main(String[] args) throws notCaptainException{
 		Game newGame = new Game();
-		newGame.addPlayer("Chris", false);
-		newGame.addPlayer("vernon", false);
-		newGame.addPlayer("Darwin", false);
-		newGame.addPlayer("Takuma", false);
-		newGame.addPlayer("Anu", false);
-		newGame.addPlayer("jaay", false);
-		newGame.addPlayer("gordon", false);
-		newGame.addPlayer("john", false);
-		newGame.addPlayer("ewan", false);
-		newGame.addPlayer("naoya", false);
-		//newGame.removePlayer();
-		System.out.println("lobby before: " + newGame.lobby.toString());
-		System.out.println("team1 before: " + newGame.team1.toString());
-		System.out.println("team2 before: " + newGame.team2.toString());
+		int i = 0;
+		while(newGame.lobby.size() != 10){
+			System.out.println("Please enter your name: ");
+			String s = in.nextLine().trim();
+			while(s.length() == 0){
+				System.out.println("Please enter a valid name: ");
+				s = in.nextLine().trim();
+			}
```

Comment:
The idea here was to get the value initially and make sure it was a valid name (ie. cant type an epmpty string for a name). Once it has the initial value it will keep checking and asking the user to redo it until the name is legit. Then it will add the player.

Reply:
my java is a bit rusty but something like

```
String s = ""'
while(!s.length()){
	System.out.println("Enter a valid name: ");			
	s = in.nextLine().trim();			
	}					
```

Reply:
This works, thanks!